go/vt/vtadmin/api.go

```diff
@@ -338,6 +340,15 @@ func (api *API) EjectDynamicCluster(key string, value any) {
 	if ok {
 		delete(api.clusterMap, key)
 	}
+
+	// Maintain order of clusters when removing dynamic cluster
+	clusters := []*cluster.Cluster{}
```

Comment:
`NewAPI` sorts these by ID, so you can use `sort.Search` (from the standard lib, not from `vtadmin/sort`) to do this in log(n) time.

Reply:
Oh sweet! Good to know